internal/metrics/prometheus.go

```diff
@@ -13,15 +13,20 @@ import (
 	"github.com/kong/kubernetes-ingress-controller/v2/internal/dataplane/deckerrors"
 )
 
+// descriptions of these metrics are found below, where their help text is set in NewCtrlFuncMetrics()
+
 type CtrlFuncMetrics struct {
-	// ConfigPushCount is a Prometheus metric with semantics defined by its help string in NewCtrlFuncMetrics().
 	ConfigPushCount *prometheus.CounterVec
 
-	// TranslationCount is a Prometheus metric with semantics defined by its help string in NewCtrlFuncMetrics().
+	ConfigPushBrokenResources *prometheus.GaugeVec
+
 	TranslationCount *prometheus.CounterVec
 
-	// ConfigPushDuration is a Prometheus metric with semantics defined by its help string in NewCtrlFuncMetrics().
+	TranslationBrokenResources prometheus.Gauge
+
 	ConfigPushDuration *prometheus.HistogramVec
+
+	ConfigPushSuccessTime *prometheus.GaugeVec
```

Comment:
I don't see this being used in this PR. Was that intentional?

Reply:
Yeah, good catch, botched copy/paste where I forgot to to update the name after pasting+misunderstanding the envtest flow. 6f691212640f57aec7219d4e4b8a2148ab91ec9e